src/site/site.xml

```diff
@@ -21,74 +21,49 @@
-         remove the need for this custom fork. This is published to
-         my end-user's GAV for ease of releasing this to maven central
-         until the upstream update happens and is released.
-         See HBASE-14785 and HBASE-21005 for more info. -->
-    <groupId>org.apache.maven.skins</groupId>
-    <artifactId>maven-fluido-skin</artifactId>
-    <version>1.12.0</version>
+    <groupId>io.github.devacfr.maven.skins</groupId>
+    <artifactId>reflow-maven-skin</artifactId>
+    <version>${reflow-maven-skin.version}</version>
   </skin>
+
   <custom>
-    <fluidoSkin>
-      <topBarEnabled>true</topBarEnabled>
-      <sideBarEnabled>false</sideBarEnabled>
-      <googleSearch>
-        <!-- The ID of the Google custom search engine to use.
-             This one searches hbase.apache.org, issues.apache.org/browse/HBASE-*,
-             and user and dev mailing list archives. -->
-        <customSearch>000385458301414556862:sq1bb0xugjg</customSearch>
-      </googleSearch>
-      <sourceLineNumbersEnabled>false</sourceLineNumbersEnabled>
-      <skipGenerationDate>true</skipGenerationDate>
-      <breadcrumbDivider>»</breadcrumbDivider>
-      <gitHub>
-        <projectId>apache/hbase</projectId>
-        <ribbonOrientation>right</ribbonOrientation>
-        <ribbonColor>red</ribbonColor>
-      </gitHub>
-    </fluidoSkin>
+    <reflowSkin>
+      <protocolRelativeURLs>true</protocolRelativeURLs>
+      <smoothScroll>true</smoothScroll>
+      <bottomNav>
+        <column>Apache HBase Project</column>
+        <column>Project Information</column>
+        <column>Documentation and API</column>
+        <column>ASF</column>
+      </bottomNav>
+      <theme>bootswatch-litera</theme>
+      <highlightJs>true</highlightJs>
+      <brand>
+        <name><![CDATA[<div class="xtoplogo"></div>]]></name>
+        <href>index.html</href>
+      </brand>
+      <navbar theme="light" background="light"></navbar>
+      <skinAttribution>false</skinAttribution>
+      <breadcrumbs>true</breadcrumbs>
+      <!-- Switch to local resources: Do NOT use CDN. -->
+      <localResources>true</localResources>
+      <absoluteResourceURL></absoluteResourceURL>
+    </reflowSkin>
   </custom>
-  <bannerLeft>
-    <name />
-    <src />
-    <href />
-    <!--
-    <name/>
-    <height>0</height>
-    <width>0</width>
--->
-  </bannerLeft>
-  <bannerRight>
-    <name>Apache HBase</name>
-    <src>images/hbase_logo_with_orca_large.png</src>
-    <href>http://hbase.apache.org/</href>
-  </bannerRight>
-  <publishDate position="none"/>
-  <version position="none"/>
   <body>
     <head>
-      <![CDATA[<meta name="viewport" content="width=device-width, initial-scale=1.0"></meta>
-      <link rel="stylesheet" href="css/bootstrap-responsive.min.css"/>
-      <link rel="stylesheet" href="css/github.min.css"/>
-      <link rel="stylesheet" href="css/site.css"/>
-      <script src="js/highlight.min.js"></script>]]>
```

Comment:
We don't need to add these anymore because these are added by the new skin by default.